doc/scapy/build_dissect.rst

```diff
@@ -2,9 +2,9 @@
 Adding new protocols
 ********************
 
-Adding new protocol (or more correctly: a new *layer*) in Scapy is very easy. All the magic is in the fields. If the 
-fields you need are already there and the protocol is not too brain-damaged, 
-this should be a matter of minutes. 
+Adding a new protocol (or more correctly: a new *layer*) in Scapy is very easy.
```

Comment:
Why did your modify this line?

Reply:
When I followed the link, I noticed the grammar nit in "Adding **a** new protocol". (I'm not a native speaker, but to me it seems the article was missing.) So I corrected it, and while I was at it, I reindented the overlong line and removed a trailing whitespace.

Would you prefer if I split that change into a separate commit? Or should I revert it entirely?

Reply:
I personally prefer to have super long lines in the doc because it keeps git history when we only change a word (instead of having to reindent the whole paragraph)

Reply:
Ok, then I'll just undo the reindentation?

Reply:
Yes please keep the long lines as it is. I missed your change because of the line breaks that you introduced.

Reply:
> Ok, then I'll just undo the reindentation?

Done.